js/pricing.js

```diff
@@ -85,16 +89,19 @@ $(
          * Changes element's disabled/enabled state.
          *
          * @param {jQuery} element the element to change the state for
+         * @param {jQuery} elementHolder the element holder to apply or hide a tooltip
          * @param {boolean} enable denotes whether the element should be enabled.
          * If `true` — enables the element, otherwise — disables
          */
-        function changeElementState(element, enable) {
+        function changeElementState(element, elementHolder, enable) {
```

Comment:
I'd rather use only `element` parameter, but pass to the function the `orderButtonHolder `. And then find the button inside.